debian/clickhouse-server.init

```diff
@@ -36,6 +36,11 @@ command -v flock >/dev/null && FLOCK=flock
 # Override defaults from optional config file
 test -f /etc/default/clickhouse && . /etc/default/clickhouse
 
+# Environment for run daemon
+if [ -f /etc/sysconfig/$PROGRAM ]; then
```

Comment:
Also we should support /etc/default for debian based systems.

Reply:
@proller also suggested that we can simply `source` (`.`) this file instead of manual parsing.